docs/hardware/devices/heltec-automation/mesh-node/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
+id: mesh-node
+title: HELTEC® Mesh Node
+sidebar_label: Mesh Node
+sidebar_position: 4
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<Tabs
+groupId="heltec"
+queryString="heltec"
+defaultValue="mesh_node_t114"
+values={[
+{label: 'Mesh Node T114', value: 'mesh_node_t114'}
+]}>
+<TabItem value="mesh_node_t114">
+
+## Mesh Node T114
+
+- **MCU:**
+  - nRF52840 (Bluetooth)
+- **LoRa Transceiver:**
+  - Semtech SX1262
+- **Frequency Options:**
+  - 433 MHz
+  - 470 - 510 MHz
+  - 863 - 870 MHz
+  - 902 - 928 MHz
+- **Connectors:**
+  - USB-C
+  - 2x1.25 lithium battery interface
+  - 2x1.25 solar panel interface
+  - 8x1.25 GNSS interface
+  - Antenna:
+    - U.FL/IPEX antenna connector for LoRa.
+    - Integrated 2.4 GHz patch antenna.
+
+### Features
+
+- Onboard 1.14 inch TFT-LCD display is optional.
+- GPS module is optional.
+- Excellent low power characteristics.
+
+### Flashing
+
+1. Trigger DFU mode manually by double-clicking RST button.
+
+2. Send or paste the prepared firmware UF2 file into the device.
+
+:::note
+
+The firmware of  no-screen version is not the same as the screen version. The password for the no-screen version defaults to '123456'. If the device prompts you incorrectly, you may have downloaded the wrong version of the firmware. If you can't find the appropriate version, please contact support@heltec.cn.
+
```

Comment:
I'd like to work with heltec to merge this into a single version. There are ways to detect if a screen is connected or not :-)

Reply:
We have this idea, but it is not finished yet, so I want to add this note temporarily in this place. But this is your document, I should not add this unstable information to someone else's document. It's my fault, I will delete it.